Makefile

```diff
@@ -167,18 +167,22 @@ help:  ## Display this help
 .PHONY: test
 test: generate lint go-test ## Run generate lint and tests
 
+KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
```

Comment:
these changes shouldn't be in here, looks like you have an extra commit from your other PR here

Reply:
Actually, I believe these changes are for another issue of this same project, this issue https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1990

Reply:
I think I have raised the 2 PRs from the same branch, maybe that might be causing the issue?

Reply:
How should I solve this?

Reply:
If both PRs are based off of one branch then I think you might need to close and reopen one. For the PR you want to keep, i.e. this one, you can back up the branch, drop the extra commit, and force push. For the other PR, you can create a new branch off main and cherry pick the extra commit from your backed up branch (w/ both changes) to there.